interfaces/apparmor/template.go

```diff
@@ -60,10 +60,19 @@ var defaultTemplate = `
   /usr/share/terminfo/** k,
   /usr/share/zoneinfo/** k,
   owner @{PROC}/@{pid}/maps k,
+  owner @{HOME}/.Private/ r,
```

Comment:
this probably merits a comment being the base template, to a casual reader the security consequences or non-consequences of this are not very clear, the bug in commit isn't clear on that either

Reply:
Done